DVSRegister.BusinessLogic/Automapper/LastModifiedDateResolver.cs

```diff
@@ -2,25 +2,32 @@
 using DVSRegister.BusinessLogic.Models.CAB;
 using DVSRegister.CommonUtility;
 using DVSRegister.Data.Entities;
+using System;
+using System.Globalization;
 
 namespace DVSRegister.BusinessLogic
 {
-    public class LastModifiedDateResolver : IValueResolver<ProviderProfile, ProviderProfileDto, string>
+    public class LastModifiedDateResolver : IValueResolver<ProviderProfile, ProviderProfileDto, DateTime>
     {
```

Comment:
if DateTime field is required instead of string, we can exclude this resolver and take CreatedTime or modified time in controller itself

Reply:
ignore this , resolver is required